extensions/sql-migration/src/constants/strings.ts

```diff
@@ -648,20 +666,53 @@ export function STATUS_VALUE(status: string, count: number): string {
 	return localize('sql.migration.status.error.count.none', "{0}", StatusLookup[status] ?? status);
 }
 
+export const MIGRATION_ERROR_DETAILS_TITLE = localize('sql.migration.error.details.title', "Migration error details");
+export const MIGRATION_ERROR_DETAILS_LABEL = localize('sql.migration.error.details.label', "Migration error(s))");
+export const OPEN_MIGRATION_DETAILS_ERROR = localize('sql.migration.open.migration.destails.error', "Error opening migration details dialog");
+export const OPEN_MIGRATION_TARGET_ERROR = localize('sql.migration.open.migration.target.error', "Error opening migration target");
+export const OPEN_MIGRATION_SERVICE_ERROR = localize('sql.migration.open.migration.service.error', "Error opening migration service dialog");
+export const LOAD_MIGRATION_LIST_ERROR = localize('sql.migration.load.migration.list.error', "Error loading migrations list");
+export const ERROR_DIALOG_CLEAR_BUTTON_LABEL = localize('sql.migration.error.dialog.clear.button.label', "Clear");
+
 export interface LookupTable<T> {
 	[key: string]: T;
 }
 
+
 export const StatusLookup: LookupTable<string | undefined> = {
-	['InProgress']: localize('sql.migration.status.inprogress', 'In progress'),
-	['Succeeded']: localize('sql.migration.status.succeeded', 'Succeeded'),
-	['Creating']: localize('sql.migration.status.creating', 'Creating'),
-	['Completing']: localize('sql.migration.status.completing', 'Completing'),
-	['Canceling']: localize('sql.migration.status.canceling', 'Canceling'),
-	['Failed']: localize('sql.migration.status.failed', 'Failed'),
+	[MigrationStatus.InProgress]: localize('sql.migration.status.inprogress', 'In progress'),
+	[MigrationStatus.Succeeded]: localize('sql.migration.status.succeeded', 'Succeeded'),
+	[MigrationStatus.Creating]: localize('sql.migration.status.creating', 'Creating'),
+	[MigrationStatus.Completing]: localize('sql.migration.status.completing', 'Completing'),
+	[MigrationStatus.Retriable]: localize('sql.migration.status.retriable', 'Retriable'),
+	[MigrationStatus.Canceling]: localize('sql.migration.status.canceling', 'Canceling'),
+	[MigrationStatus.Canceled]: localize('sql.migration.status.canceled', 'Canceled'),
```

Comment:
I notice we use 'Canceled' here, but 'Cancelled' (two Ls) in other places like the legacy status codes - presumably we'll use the spelling with one L everywhere moving forward?

Reply:
yes, I added for legacy references.  It shows up inconsistently at times.